tests/Unit/ValueObject/DateValueObjectTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Unit\Patterns;
+
+use Drupal\oe_theme\ValueObject\DateValueObject;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Test date value object.
+ */
+class DateValueObjectTest extends UnitTestCase {
+
+  /**
+   * Test constructing a date value object from an array.
+   */
+  public function testFromArray() {
```

Comment:
If we are testing the fromArray method I guess testing the other methods (fromTimestamp, from DateTimePlus) is also a good idea

Reply:
Ok

Reply:
I removed the methods from the DateValueObject because they are no more relevant for now.